content/sensu-go/5.17/installation/auth.md

```diff
@@ -619,7 +631,7 @@ For example:
 
 ## AD authentication
```

Comment:
```suggestion
## ActiveDirectory authentication {#ad-authentication}
```

optional suggestion -- not sure if this introduces too much out-of-scope change but this sort of bothered me